cl/opinion_page/views.py

```diff
@@ -563,6 +583,28 @@ async def make_thumb_if_needed(
     return rd
 
 
+async def download_docket_entries_csv(
+    request: HttpRequest, docket_id: int
+) -> HttpResponse:
+    """Download csv file containing list of DocketEntry for specific Docket"""
+
+    docket, _ = await core_docket_data(request, docket_id)
+    de_list = await fetch_docket_entries(docket)
+    court_id = docket.court_id
+    case_name = docket.slug
+
+    date_str = datetime.datetime.now().strftime("%Y-%m-%d")
+    filename = f"{case_name}.{court_id}.{docket_id}_{date_str}.csv"
```

Comment:
I think we wanted these to be dot-separated, so:

```suggestion
    filename = f"{case_name}.{court_id}.{docket_id}.{date_str}.csv"
```

Reply:
Yes! fixed